vendor/assets/stylesheets/components/modal.scss

```diff
@@ -0,0 +1,25 @@
+// for some reason these are missing from uikit 2 even though the styles are in the docs
```

Comment:
No need for this comment. The default uikit theme has minimal styling. Just treat these as custom overrides the way you would for any other custom uikit styling.